python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -1266,6 +1305,16 @@ def main(args):
         convert_safety_checker(pipe, args)
         logger.info("Converted safety_checker")
 
+    if args.convert_unet and args.refiner_version is not None:
+        logger.info(f"Converting refiner")
+        del pipe
+        gc.collect()
+        pipe = get_pipeline(args.refiner_version)
```

Comment:
This is a great idea to avoid the extra download. I had experimented with simply deleting the extra parts of the pipeline too, but as you mentioned since we need the text encoder config, we'd only be able to delete the vae. If everyone likes the idea of starting to save the config files into the output folder, this could be pretty doable.